spring-security/src/test/java/com/sap/cloud/security/spring/config/ConfigurationUtil.java

```diff
@@ -0,0 +1,16 @@
+package com.sap.cloud.security.spring.config;
+
+import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.config.ServiceConstants;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ConfigurationUtil {
```

Comment:
I would move this method it to the XsuaaServiceConfigurationTest class as it's the only place where it is used.

Reply:
Well, not really - it has been our discussion before that it is also used here https://github.com/SAP/cloud-security-services-integration-library/pull/1393/commits/b93553697ac9b1cadae98cb4652da74f062483ea#diff-f1efe439764be3d82df836e42d26dd47e909646b0ea7c00a4fe8d375b4766de6
(XsuaaServiceConfigurationTest  and XsuaaServiceConfiguration**s**Test )